README.md

```diff
@@ -1,13 +1,13 @@
 # Cycle Music
-![Screencapture](https://pbs.twimg.com/media/DBAOyg2XoAAUDjl.jpg:large)
+![Screencapture](https://i.goopics.net/01nxw.png)
```

Comment:
Maybe we should capture the application without the event name

Reply:
yes, for master only ?

Reply:
Yes

Reply:
:ok_hand: